src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -3,16 +3,18 @@ import GeLabelService from 'c/geLabelService';
 import { fireEvent } from 'c/pubsubNoPageRef';
 import TemplateBuilderService from 'c/geTemplateBuilderService';
 import getOrgDomain from '@salesforce/apex/GE_GiftEntryController.getOrgDomain';
+import { format } from 'c/utilCommon';
+// TODO: maybe import data import token field reference?
```

Comment:
Should this todo stay or can it be removed?

Reply:
Doesn't appear we need that anymore. Removed